### PR TITLE
use new travelnet api

### PR DESCRIPTION
**Pull request by BuckarooBanzay**
See: https://github.com/mt-mods/travelnet/issues/69

Important changes:
* removed `find_travelnet_from_db` metadata-restoration function because iterating over _all_ players in mod-storage isn't really feasible IMO

CI-Error:
`Error: Resource not accessible by integration` :shrug:

### CODE REVIEW

**Comment by S-S-X** [comment]
Yeah, seems like that github comment thing is broken but actual tests are passing.
Actually there's prob no tests at all for travelnet thing anyway...

I think most of travelnet stuff could prob be removed anyway, long since I last heard about issues where this would be needed.
I guess handout to / claim from shared account might be only thing that's useful nowdays.

Privileged tool so if it crashes server because travelnet mod wasn't updated I can just say it was admin's fault 😂

**Comment by S-S-X** [comment]
Well, there's magic pen which isn't privileged but I guess I can add check / backwards compat for that if needed, hopefully not needed.

**Comment by S-S-X** [comment]
> removed find_travelnet_from_db metadata-restoration function because iterating over all players in mod-storage isn't really feasible IMO

Btw, these were all like that because it was intentionally checking whole db. There was invalid entries in db where travelnet was under wrong position in database. Probably can still be done for example by pushing travelnet, replacing it with constructor or something similar.
Mixing travelnet and elevator networks also allow doing similar things which in turn allowed taking over tavelnet location in different players network.

Basically situation is that travelnet metadata and position does not match with database entry in any way but link stays because travelnet mod wasn't double checking for travel.
